metrix/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// pub mod schema;
+// pub mod models;
+
+// #[macro_use] extern crate diesel;
```

Comment:
this was all shit from the diesel tutorial... https://diesel.rs/guides/getting-started/

I'm leaving it in for now, maybe clean up once the schema is hooked up and we're actually querying.